app/generate_command_metadata.py

```diff
@@ -54,6 +54,10 @@
     "IDC_BOOKMARK_BAR_UNTRACK_PRICE_FOR_SHOPPING_BOOKMARK",
     "IDC_CHROME_MENU",
     "IDC_MEDIA_TOOLBAR_CONTEXT_REPORT_CAST_ISSUE",
+    "IDC_AUTOFILL_MANDATORY_REAUTH",
+    "IDC_PASSWORDS_AND_AUTOFILL_MENU",
+    "IDC_SHOW_SEARCH_COMPANION",
+    "IDC_SHOW_CHROME_LABS",
```

Comment:
I *think* this should look like this:
```
@@ -54,6 +54,7 @@ EXCLUDE_COMMANDS = [
     "IDC_BOOKMARK_BAR_UNTRACK_PRICE_FOR_SHOPPING_BOOKMARK",
     "IDC_CHROME_MENU",
     "IDC_MEDIA_TOOLBAR_CONTEXT_REPORT_CAST_ISSUE",
+    "IDC_SHOW_SEARCH_COMPANION",

     # ChromeOS only
     "IDC_TOGGLE_REQUEST_TABLET_SITE",
@@ -168,6 +169,9 @@ EXISTING_TRANSLATIONS = {
     "IDC_OPEN_GUEST_PROFILE": "IDS_OPEN_GUEST_PROFILE",
     "IDC_SHOW_BRAVE_WEBCOMPAT_REPORTER": "IDS_SHOW_BRAVE_WEBCOMPAT_REPORTER",
     "IDC_COPY_CLEAN_LINK": "IDS_COPY_CLEAN_LINK",
+    "IDC_SHOW_CHROME_LABS": "IDS_CHROMELABS",
+    "IDC_AUTOFILL_MANDATORY_REAUTH": "IDS_AUTOFILL_MANDATORY_REAUTH_ICON_TOOLTIP",
+    "IDC_PASSWORDS_AND_AUTOFILL_MENU": "IDS_PASSWORDS_AND_AUTOFILL_MENU",
 }
```

Reply:
Thank you, I missed that! Fixed in afe8cdcf09c54f16837c76d0ef6276812c2a065d.